root/programs/C#/Frameworks/Infrastructure/CustomControl/PropertyGrid/CheckType.cs

```diff
@@ -64,21 +64,21 @@ public CheckType()
         #region プロパティ
 
         /// <summary>必須</summary>
-        private bool _isIndispensabile;
+        private bool _isIndispensable;
 
         /// <summary>必須</summary>
         [DefaultValue(false), NotifyParentProperty(true),
         RefreshProperties(RefreshProperties.Repaint)]
-        public bool IsIndispensabile
+        public bool IsIndispensable
```

Comment:
@daisukenishino 

What do you want to say?
Please explain this sentence to me.